packages/appstore/src/components/add-more-wallets/carousel-container.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import useEmblaCarousel, { EmblaCarouselType, EmblaOptionsType } from 'embla-carousel-react';
+import { PrevButton, NextButton } from './carousel-buttons';
+import { observer, useStore } from '@deriv/stores';
+import { DesktopWrapper, MobileWrapper } from '@deriv/components';
+
+const CarouselContainer: React.FC<{ children: React.ReactNode }> = observer(({ children }) => {
+    const {
+        ui: { is_mobile },
+    } = useStore();
+
+    const options: EmblaOptionsType = {
+        align: 0,
+        containScroll: 'trimSnaps',
+        watchDrag: !is_mobile && false,
```

Comment:
@aizad-deriv mmm seems like `watchDrag` will be always false, no matter is_mobile true or not🤔

Reply:
Looks like it @george-usynin-deriv for pointing that out will make the changes accordingly 🙏🏼